openwisp_users/api/urls.py

```diff
@@ -9,6 +10,24 @@ def get_api_urls(api_views=None):
     urlpatterns = []
     if api_views is None:
         api_views = views
+    urlpatterns += [
+        path('users/organization/', views.organization_list, name='organization_list',),
+        path(
+            'users/organization/<str:pk>/',
+            views.organization_detail,
+            name='organization_detail',
+        ),
+        path('users/user/', views.user_list, name='user_list',),
+        path('users/user/<str:pk>/', views.user_detail, name='user_detail'),
```

Comment:
We are showing the hashed password, do we want to send it back like this @nemesisdesign @ManishShah120 ? I would assume we wouldn't share this in API even though the UI shows it; I don't know in which case sharing the password would be helpful! 
![ss4](https://user-images.githubusercontent.com/24985760/128630189-45244f94-cbe6-468d-bdaa-32135749ba49.png)

Reply:
> Create a new user with organizations value.
> Open the user with UUID in API UI, you'll notice while the value is present in json, form is empty:

Yes, @atb00ker, the form is present to add or remove org user(if the sent data is same then it will get removed else, the changes will be accomplished, this field would have been populated if there would be m2m relationship, for example what would happen if the user is OrgUser of more than one Org?? :smile:

Reply:
Org1admin cannot see organization user's section!
ss4

While an org_users connection exists :

            "organization_users": [
                {
                    "is_admin": false,
                    "organization": "091dbab0-4188-4e83-a2e3-40853c9189f4"
                }
            ]
Yes, this is intentinal, the Org1admin, can only see the the org_users, present but cannot edit anything, so I removed this field for it.

Reply:
> Create an org owner and try to downgrade it, not handled gracefully.

@atb00ker This is already been handled by the `full_clean` method, so will it be not redundant to repeat this, in usecase/production, it will just return the error. I guesss??

Reply:
> @atb00ker This is already been handled by the full_clean method, so will it be not redundant to repeat this, in usecase/production, it will just return the error in normal way. I guess this is what we did in the config App!!!

I belive the response should be graceful when possible. Unknown cases can happily return 500, but a known case should give error message and explain the message! 😄

Reply:
> Yes, this is intentinal, the Org1admin, can only see the the org_users, present but cannot edit anything, so I removed this field for it.

If the user is not allowed to edit something, we should have it as readonly for consistency, what do you think? (Also, this can be lower priority)